third_party/go/bazel_differ_deps.bzl

```diff
@@ -0,0 +1,138 @@
+# This file was manually copied from https://github.com/ewhauser/bazel-differ/blob/916e6409ae501c56ac4d6da9c0785c2ee97721b3/deps.bzl because gazelle doesn't seem to like working with dep files from remote repositories.
```

Comment:
I've had this issue recently as well, was it related to `select` clauses in generated BUILD files?
https://github.com/aspect-build/bazel-examples/pull/4#issue-1222210613

One of us should file upstream on gazelle...